lib/apiservers/engine/backends/container_proxy.go

```diff
@@ -709,6 +709,9 @@ func (c *ContainerProxy) AttachStreams(ctx context.Context, vc *viccontainer.Vic
 func dockerContainerCreateParamsToPortlayer(cc types.ContainerCreateConfig, layerID string, imageStore string) *containers.CreateParams {
 	config := &models.ContainerCreateConfig{}
 
+	config.NumCpus = &cc.HostConfig.CPUCount
+	config.Memory = &cc.HostConfig.Memory
```

Comment:
Suggest using MemoryMB, as that's what spec uses (https://www.vmware.com/support/developer/vc-sdk/visdk25pubs/ReferenceGuide/vim.vm.ConfigSpec.html)